ios/browser/api/bookmarks/exporter/bookmark_html_writer.cc

```diff
@@ -497,16 +498,23 @@ void BookmarkFaviconFetcher::ExecuteWriter() {
   // BookmarkModel isn't thread safe (nor would we want to lock it down
   // for the duration of the write), as such we make a copy of the
   // BookmarkModel using BookmarkCodec then write from that.
+  bookmarks::BookmarkModel* bookmark_model_;
+  if (base::FeatureList::IsEnabled(
```

Comment:
do we actually want this enabled or should we override this to disable by default?

Reply:
I wonder why it's only enabled on ios for now

Reply:
cc: @kylehickinson

Reply:
in particular I'm wondering if there are any service side changes required here. This probably needs QA attention if we decide to keep the flag enabled

Reply:
As discussed this is also enabled on Android as well, I believe we are still waiting to hear back on whether or not we need sync server changes to support it